nativelib/src/main/scala/scala/scalanative/runtime/package.scala

```diff
@@ -119,6 +119,11 @@ package object runtime {
    */
   @noinline def loop(): Unit = ExecutionContext.loop()
 
+  /** Run the runtime's event loop once and return true when internal queue has
+   *  more Runnable to run.
+   */
+  @noinline def loopRunOnce(): Boolean = ExecutionContext.loopRunOnce()
```

Comment:
Personally I think we should not introduce new methods to `runtime` package especially since QueueExecutionContext would now probably be a niche in context of having a real multithreading. If so, it would be better to add this methods to `runtime.ExecutionContext`.